cmd/controller/controller.go

```diff
@@ -225,12 +225,23 @@ func (c *CmdOpts) startController() error {
 	// common factory to get the admin kube client that's needed in many components
 	adminClientFactory := kubernetes.NewAdminClientFactory(c.K0sVars)
 
+	enableKonnectivity := true
```

Comment:
inverted could be a little easier to read

like this (though I don't like a lot of inversions here)

```
var enableKonnectivity bool // false by default
if !c.SingleNode && !c.EnableWorker{
enableKonnectivity = !stringslice.Contains(c.DisableComponents, constant.KonnectivityServerComponentName)
}
```

Reply:
Hi @soider, in above code snippet, if  `--enable-worker` is true, `enableKonnectivity` will always be false, this is not expected. In addition, if `!c.SingleNode && !c.EnableWorker`,  `enableKonnectivity` should always be true, that's one of purposes of this PR.